docssrc/src/commandtrees.md

```diff
@@ -80,6 +80,108 @@ In this example, we have our normal `/sayhi` command using the `executes()` meth
 
 </div>
 
+### Reduce indentation with nested arguments
+
+Sometimes we will need such a `CommandTree`:
+
+<div class="multi-pre">
+
+```java,Java
+new CommandTree("example")
+    .then(new LiteralArgument("arg1")
+    .then(new StringArgument("arg2")
```

Comment:
This needs one more level of indentation

Reply:
oh thanks! i do not notice this at all xD